lib/app/features/wallets/pages/edit_wallet_modal/edit_wallet_modal.dart

```diff
@@ -65,10 +70,7 @@ class EditWalletModal extends HookConsumerWidget {
               child: isNameChanged
                   ? Button(
                       onPressed: () {
-                        ref
```

Comment:
Maybe we need to keep methods like `addWallet' and `updateWallet` but with an empty body if you need another implementation of those methods and not remove them from the UI.

Reply:
Discussed, resolving.